src/functions/public/Auth/Revoke-GitHubToken.ps1

```diff
@@ -0,0 +1,136 @@
+﻿function Revoke-GitHubToken {
```

Comment:
I want to change the name of the function (and thus the file) to be `Revoke-GitHubAccessToken`.